### PR TITLE
Get biorxiv tweet count

### PR DESCRIPTION
**Pull request by lightninglu10**
Ping the API to get biorxiv tweet count

### CODE REVIEW

File: src/paper/tasks.py [hunk 1]
```diff
@@ -995,6 +995,24 @@ def pull_biorxiv(page=0, retry=0):
...
+        )
+        unescaped_json_res = res.text
+        # For some reason, the request data comes back with "\ufeff" and then also has an open paren ( and a closed paren )
+        escaped_json_res = json.loads(res.text.strip("\ufeff(").rstrip(res.text[-1]))
+        count = 0
+        for message in escaped_json_res["messages"]:
+            count += message["count_tweets"] + message["count_retweets"]
+        return count
+    except Exception as e:
+        print(e)
```
Comment (le-sun) [inline src/paper/tasks.py:1011]: Delete

File: src/paper/tasks.py [hunk 2]
```diff
@@ -995,6 +995,24 @@ def pull_biorxiv(page=0, retry=0):
         sentry.log_error(e)
 
 
+def _get_biorxiv_tweet_counts(url, doi):
+    try:
+        res = requests.get(
```
Comment (le-sun) [inline src/paper/tasks.py:1000]: Let's add a timeout

**Review by le-sun — CHANGES_REQUESTED** [review]
Couple small changes

File: src/paper/tasks.py (hunk 1 above)
Comment (lightninglu10) [reply src/paper/tasks.py:1011]: I'd like to keep it in here, otherwise during development these errors are swallowed up

Comment (lightninglu10) [reply src/paper/tasks.py:1011]: better to print it to the console IMO

**Pull request merged**